app/app/local_settings.py.dist

```diff
@@ -92,6 +92,7 @@ GITHUB_CLIENT_ID = 'TODO'
 GITHUB_CLIENT_SECRET = 'TODO'
 GITHUB_API_USER = 'TODO'
 GITHUB_API_TOKEN = 'TODO'
+GITHUB_APP_NAME = 'TODO'
```

Comment:
GITHUB_API_BASE_URL
GITHUB_SCOPE

these should probably go in the .dist, no?

Reply:
GITHUB_APP_NAME too

Reply:
I was considering `GITHUB_SCOPE`, `GITHUB_API_BASE_URL`, and `GITHUB_TOKEN_URL` to be more repo specific than local dev specific. Realistically, I could eliminate the api/token urls since they shouldn't change.  Scope would only change if the project as a whole was changing to require more permissions from the user (so a dev would want to commit that change in order to allow any added functionality to work for everyone).  Do you want me to move them?